libs/s25main/nodeObjs/noSign.cpp

```diff
@@ -16,8 +16,12 @@
 // along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
 
 #include "noSign.h"
+#include "../GlobalGameSettings.h"
+#include "../world/GameWorldGame.h"
```

Comment:
Such "go up" includes should never be required. Can you try this (needs resorting the include):

```suggestion
#include "GlobalGameSettings.h"
#include "world/GameWorldGame.h"
```